pkgs/top-level/all-packages.nix

```diff
@@ -4127,6 +4127,8 @@ in
 
   jumanpp = callPackage ../tools/text/jumanpp {};
 
+  jump = callPackage ../tools/system/jump {};
```

Comment:
I'm not married to where I placed it, so if anyone has a better suggestion I'm all ears.